src/core/data-abbr.js

```diff
@@ -0,0 +1,44 @@
+// @ts-check
+// Module core/data-abbr
+// - Finds all elements with data-abbr attribute and processes them.
+import { showInlineWarning } from "./utils";
+export const name = "core/dfn-abbr";
+
+export function run() {
+  /** @type {NodeListOf<HTMLElement>} */
+  const elements = document.querySelectorAll("[data-abbr]");
+  for (const elem of elements) {
```

Comment:
```elements.forEach``` should work well now, if it's on a different line.
Don't need the type casting above as well.

Reply:
Decided `for` was nicer here... no benefit of using forEach.